dimod/typing.py

```diff
@@ -39,15 +39,16 @@
 
 __all__ = ['Bias',
            'GraphLike',
+           'Polynomial',
            'SampleLike',
            'SamplesLike',
            'Variable',
            'VartypeLike',
            ]
 
-
-Variable = Hashable  # todo: exclude None
 Bias = Union[int, float, np.number]
+Variable = Hashable  # todo: exclude None
```

Comment:
Why move this? Unless it's to make it alphabetical in which case why put Polynomial after?

Also, adding `Polynomal` needs a release note.

Reply:
It was to make it alphabetical but `Variable` is required for `Polynomial`, so best-effort alphabetical. 
Will add RN
(I just rebased too)